### DISCUSSION TITLE
Table/Grid Support?

### DISCUSSION THREAD

**Discussion by rijans** [vakata/jstree#2532] [Q&A] [answered]
I really love jsTree. But I couldn't find a proper stable Table/Grid support for/from jsTree as I needed in a WebApp. Some user implementations are years old, and no more working. Is there any official Table-Grid support for jsTree?

Please help

**Comment by vakata** [comment, marked as answer]
No, there is no built-in grid. There was a good plugin a few years back: https://github.com/deitch/jstree-grid but I see that it is no longer maintained. Since there are no breaking API changes it should work with any recent jstree version.
